core/src/main/java/org/pentaho/di/core/row/ValueDataUtil.java

```diff
@@ -1813,26 +1944,53 @@ public static final boolean onlySpaces( String str ) {
    * @param dataA
    *          The value (filename)
    * @return true if the file is well formed.
+   * @deprecated Use {@link ValueDataUtil#isXMLFileWellFormed(ValueMetaInterface, Object, boolean)} instead
    */
+  @Deprecated
   public static boolean isXMLFileWellFormed( ValueMetaInterface metaA, Object dataA ) {
+    boolean xmlWellFormed = false;
+    try {
+      xmlWellFormed = isXMLFileWellFormed( metaA, dataA, false );
+    } catch ( KettleFileNotFoundException e ) {
+      // Ignore
+    }
+    return xmlWellFormed;
+  }
+
+  /**
+   * Checks an xml file is well formed.
+   *
+   * @param metaA
+   *          The ValueMetaInterface
+   * @param dataA
+   *          The value (filename)
+   * @param failIfNoFile
+   *          Indicates if the transformation should fail if no file is found
+   * @return true if the file is well formed.
+   * @throws KettleFileNotFoundException
+   */
+  public static boolean isXMLFileWellFormed( ValueMetaInterface metaA, Object dataA, boolean failIfNoFile )
+          throws KettleFileNotFoundException {
     if ( dataA == null ) {
+      if ( failIfNoFile ) {
```

Comment:
This actually is not a file not found.... it's a file path/name not provided.... I'm not sure the file not found exception suits this case